include/unifex/create.hpp

```diff
@@ -0,0 +1,196 @@
+
+  private:
+    friend void tag_invoke(tag_t<start>, type& self) noexcept try {
+      self.fn_(self);
+    } catch(...) {
+      unifex::set_error((Receiver&&) self.rec_, std::current_exception());
+    }
+
+    // Forward other receiver queries
+    template (typename CPO)
+      (requires is_receiver_query_cpo_v<CPO> AND is_callable_v<CPO, const Receiver&>)
+    friend auto tag_invoke(CPO cpo, const type& self)
+        noexcept(is_nothrow_callable_v<CPO, const Receiver&>)
+      -> callable_result_t<CPO, const Receiver&> {
+      return std::move(cpo)(self.rec_);
+    }
+
+    UNIFEX_NO_UNIQUE_ADDRESS Receiver rec_;
+    UNIFEX_NO_UNIQUE_ADDRESS Fn fn_;
+    UNIFEX_NO_UNIQUE_ADDRESS Context ctx_;
+  };
+};
+
+template <typename Receiver, typename Fn, typename Context>
+using _operation = typename _op<Receiver, Fn, Context>::type;
+
+template <typename Fn, typename Context>
+struct _snd_base {
+  struct type {
+    template <template<typename...> class Variant>
+    using error_types = Variant<std::exception_ptr>;
+
+    static constexpr bool sends_done = false;
+
+    template (typename Self, typename Receiver)
+      (requires derived_from<remove_cvref_t<Self>, type> AND
+        constructible_from<Fn, member_t<Self, Fn>>)
+    friend _operation<remove_cvref_t<Receiver>, Fn, Context>
+    tag_invoke(tag_t<connect>, Self&& self, Receiver&& rec)
+        noexcept(std::is_nothrow_constructible_v<
+          _operation<Receiver, Fn, Context>,
+          member_t<Self, Fn>,
+          member_t<Self, Context>,
+          Receiver>) {
+      return _operation<remove_cvref_t<Receiver>, Fn, Context>{
+        (Receiver&&) rec,
+        (member_t<Self, Fn>&&) self.fn_,
+        (member_t<Self, Context>&&) self.ctx_};
+    }
+
+    UNIFEX_NO_UNIQUE_ADDRESS Fn fn_;
+    UNIFEX_NO_UNIQUE_ADDRESS Context ctx_{};
+  };
+};
+
+template <typename Fn, typename Context, typename... ValueTypes>
+struct _snd {
+  struct type : _snd_base<Fn, Context>::type {
+    template <template<typename...> class Variant, template <typename...> class Tuple>
+    using value_types = Variant<Tuple<ValueTypes...>>;
+  };
+};
+
+template <typename Fn, typename... ValueTypes>
+using _sender = typename _snd<Fn, detail::_empty<>, ValueTypes...>::type;
+
+template <typename Fn, typename Context, typename... ValueTypes>
+using _sender_with_context = typename _snd<Fn, Context, ValueTypes...>::type;
+
+template <typename T>
+T void_cast(void* pv) noexcept {
+  return static_cast<T&&>(*static_cast<std::add_pointer_t<T>>(pv));
+}
+
+namespace _cpo {
+template <typename... ValueTypes>
+struct _fn {
+  template (typename Fn)
+    (requires move_constructible<Fn>)
+  _sender<Fn, ValueTypes...> operator()(Fn fn) const
+      noexcept(std::is_nothrow_constructible_v<_sender<Fn, ValueTypes...>, Fn>) {
+    return _sender<Fn, ValueTypes...>{(Fn&&) fn};
+  }
+  template (typename Fn, typename Context)
+    (requires move_constructible<Fn> AND move_constructible<Fn>)
```

Comment:
You've duplicated `move_constructible<Fn>`; is the second one supposed to be `move_constructible<Context>`?
```suggestion
    (requires move_constructible<Fn> AND move_constructible<Context>)
```

Reply:
Nice spot.